### PR TITLE
Redirect /owners to /settings

### PR DESCRIPTION
**Pull request by hi-rustin**
close https://github.com/rust-lang/crates.io/issues/3244

### CODE REVIEW

**Comment by hi-rustin** [comment]
r? @locks 

I don't know if that's what you mean.

File: app/routes/crate/owners.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class OwnersRoute extends Route {
-  setupController(controller) {
-    super.setupController(...arguments);
+  @service notifications;
+  @service redirector;
```
Comment (Turbo87) [inline app/routes/crate/owners.js:6]: these service don't appear to be used. was that intentional?

File: tests/acceptance/settings-test.js [hunk 2]
```diff
@@ -8,13 +8,13 @@ import { setupApplicationTest } from 'cargo/tests/helpers';
 
 import axeConfig from '../axe-config';
 
-module('Acceptance | /crates/:name/owners', function (hooks) {
+module('Acceptance | /crates/:name/settings', function (hooks) {
```
Comment (Turbo87) [inline tests/acceptance/settings-test.js:11]: can you add a test that checks that the redirection from `/crates/:name/owners` to `/crates/:name/settings` works correctly? 🙏

File: app/routes/crate/owners.js (hunk 1 above)
Comment (hi-rustin) [reply app/routes/crate/owners.js]: Removed.

File: tests/acceptance/settings-test.js (hunk 2 above)
Comment (hi-rustin) [reply tests/acceptance/settings-test.js:11]: Added.

**Comment by Turbo87** [comment]
I've slightly simplified the redirection test, but other than that this looks great. Thanks!

@bors r+

**Pull request merged**